esbuild-plugin-metadata-injector/placeholder.js

```diff
@@ -0,0 +1,7 @@
+// @ts-check
+
+import gitDescribe from "git-describe";
+import packageJSON from "../package.json" with { type: "json" };
+
+export const PACKAGE_VERSION = packageJSON.version;
+export const GIT_DESCRIBE = gitDescribe.gitDescribeSync().raw;
```

Comment:
コンテナイメージで常にdirtyと判断されてしまうのでdirty markは表示しないようにする
```suggestion
export const GIT_DESCRIBE = gitDescribe.gitDescribeSync({
  // HACK: remove dirty mark
  customArguments: ["--dirty="],
}).raw;
```

Reply:
`dirtyMark: ""`では？

Reply:
> `dirtyMark: ""`では？

無理
https://discord.com/channels/391390986770710528/391392898853371915/1249602462868832256

Reply:
Dockerfile ARGから流し込むようにして、これはやめよう